public/app/features/alerting/unified/components/settings/ExternalAlertmanagers.tsx

```diff
@@ -0,0 +1,126 @@
+import { WithReturnButton } from '../WithReturnButton';
+
+import { useSettings } from './SettingsContext';
+
+interface Props {
+  onEditConfiguration: (dataSourceName: string) => void;
+}
+
+export const ExternalAlertmanagers = ({ onEditConfiguration }: Props) => {
+  const {
+    externalAlertmanagerDataSourcesWithStatus: externalAlertmanagersWithStatus,
+    deliverySettings,
+    enableAlertmanager,
+    disableAlertmanager,
+  } = useSettings();
+
+  // determine if the alertmanger is receiving alerts
+  // this is true if Grafana is configured to send to either "both" or "external" and the Alertmanager datasource _wants_ to receive alerts.
+  const isReceivingOnAlertmanager = (
+    externalDataSourceAlertmanager: ExternalAlertmanagerDataSourceWithStatus
+  ): boolean => {
+    const sendingToExternal = [AlertmanagerChoice.All, AlertmanagerChoice.External].some(
+      (choice) => deliverySettings?.alertmanagersChoice === choice
+    );
+    const wantsAlertsReceived = isAlertmanagerDataSourceInterestedInAlerts(
+      externalDataSourceAlertmanager.dataSourceSettings
+    );
+
+    return sendingToExternal && wantsAlertsReceived;
+  };
+
+  return (
+    <Stack direction="column" gap={0}>
+      {externalAlertmanagersWithStatus.map((alertmanager) => {
+        const { uid, name, jsonData, url } = alertmanager.dataSourceSettings;
+        const { status } = alertmanager;
+
+        const isReceiving = isReceivingOnAlertmanager(alertmanager);
+        const provisionedDataSource = alertmanager.dataSourceSettings.readOnly === true;
+        const detailHref = createUrl(DATASOURCES_ROUTES.Edit.replace(/:uid/gi, uid));
+        const readOnlyDataSource =
+          provisionedDataSource || isVanillaPrometheusAlertManagerDataSource(alertmanager.dataSourceSettings.name);
+
+        const handleEditConfiguration = () => onEditConfiguration(name);
+
+        return (
+          <Card key={uid}>
+            <Card.Heading>
+              <Stack alignItems="center" gap={1}>
+                <WithReturnButton title="Alerting settings" component={<TextLink href={detailHref}>{name}</TextLink>} />
+                {provisionedDataSource && <ProvisioningBadge />}
+              </Stack>
+            </Card.Heading>
+            <Card.Figure>
+              <img alt="Alertmanager logo" src="public/app/plugins/datasource/alertmanager/img/logo.svg" />
+            </Card.Figure>
+
+            <Card.Meta>
+              {capitalize(jsonData.implementation ?? 'Prometheus')}
+              {url}
+            </Card.Meta>
+
+            <Card.Description>
+              {!isReceiving ? (
+                <Text variant="bodySmall">Not receiving Grafana-managed alerts</Text>
+              ) : (
+                <>
+                  {status === 'pending' && <Badge text="Activation in progress" color="orange" />}
+                  {status === 'active' && <Badge text="Receiving Grafana-managed alerts" color="green" />}
```

Comment:
```suggestion
                  {status === 'active' && <Badge text="Receiving Grafana managed alerts" color="green" />}
```

(I only suggest this because it seems we refer to it as `Grafana managed` elsewhere from what I could see)

Reply:
Oh good catch! I'll check with Brenda on our preferred notation :D